proto/decentraland/sdk/components/player_clicked.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+
+import "decentraland/common/vectors.proto";
+import "decentraland/sdk/components/common/id.proto";
+
+option (common.ecs_component_id) = 1107;
+
+// The PlayerClicked component is added to the RootEntity every time the user clicks on an avatar.
+// GOS component
+message PBPlayerClicked {
```

Comment:
Wouldn't be appropriate to use `PointerEventsResult` here?

Reply:
how ? The player/avatar doesnt have a collider, i dont have that data

Reply:
not needed for now. we can assume every player avatar has a collider by default, and a `PointerEvent` with an `onPointerDown`, in the future we can extend the ADR to add the other behaviour (like onHover, onLeave) by adding the specific `PointerEvent`